checks/evaluation/pinned_dependencies_test.go

```diff
@@ -852,35 +424,6 @@ func stringAsPointer(s string) *string {
 	return &s
 }
 
-func Test_generateOwnerToDisplay(t *testing.T) {
-	t.Parallel()
-	tests := []struct { //nolint:govet
-		name        string
-		gitHubOwned bool
-		want        string
-	}{
```

Comment:
this test could probably stay if `generateOwnerToDisplay` is still in the code?

Reply:
Referencing https://github.com/ossf/scorecard/pull/3829#discussion_r1479946774

Reply:
Added here: https://github.com/ossf/scorecard/pull/3829/commits/153fb4ea71319542114f99871dfe9b436e7ead9d